Jenkinsfile

```diff
@@ -18,6 +18,7 @@
 // Then a second PR submitted to comment out the @Library line, and when it
 // is landed, both PR branches can be deleted.
 //@Library(value='pipeline-lib@my_branch_name') _
+@Library(["pipeline-lib@grom72/SRE-2453", "trusted-pipeline-lib@grom72/SRE-2453"]) _
```

Comment:
This of course needs reverting before landing.

Reply:
Done